shared/modules/logger_module.go

```diff
@@ -15,7 +15,9 @@ type LoggerModule interface {
 	// TODO(#288): Embed `ObservableModule` inside of the `Module` interface so each module has its own context specific logger
 	ObservableModule
 
-	// `CreateLoggerForModule` returns the logger with additional context (e.g. module name)
-	// https://github.com/rs/zerolog#sub-loggers-let-you-chain-loggers-with-additional-context
-	CreateLoggerForModule(string) Logger
+	// CreateLoggerForModule returns the logger with additional context (e.g. module name)
```

Comment:
Do we need the same comment both next to the interface and the implementation?

Don't have a good answer to this, just feels repetative.

Reply:
I was similarly uncertain and defaulted to verbosity.

Reply:
Wdyt about leaving the on the interface and the making the comment on the implementation something like this:
```
// CreateLoggerForModule implements the `modules.Logger` interface member.
```

Reply:
I LIKE IT!

Throw it in the code review guideline ;)